### PR TITLE
Add Release notes document

### PR DESCRIPTION
**Pull request by avinash-palleti**
Adding Release document to point to release notes

Issue: #21

### CODE REVIEW

File: docs/release/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+| Apr-21, 2017 | v1.3.1 | 01.00.12 | v1.3.1 | [Release Notes](https://github.com/intel-aero/meta-intel-aero/releases/tag/v1.3.1)|
+| Apr-18, 2017 | v1.3 | 01.00.12 | v1.3 | [Release Notes](https://github.com/intel-aero/meta-intel-aero/releases/tag/v1.3)|
+| Apr-13, 2017 | v1.2 | 01.00.12 | v1.2 | [Release Notes](https://github.com/intel-aero/meta-intel-aero/releases/tag/v1.2)|
+| Dec-20, 2016 | v01.00.04 | 01.00.12 | v01.00.04| [Release Notes](https://github.com/intel-aero/meta-intel-aero/releases/tag/v01.00.04)|
+| Nov-01, 2016 | v01.00.03 | 01.00.11 | v01.00.03 | [Release Notes](https://github.com/intel-aero/meta-intel-aero/releases/tag/v01.00.03)|
+| Oct-11, 2016 | v01.00.02 | 01.00.09 | v01.00.02 | [Release Notes](https://github.com/intel-aero/meta-intel-aero/releases/tag/v01.00.02)|
+| Aug-18, 2016 | v01.00.01 | 01.00.03 | v01.00.01 | [Release Notes](https://github.com/intel-aero/meta-intel-aero/releases/tag/v01.00.01)|
+
+Corresponding files from [Download Center](https://downloadcenter.intel.com/download/26932/Intel-Aero-Platform-for-UAVs-Installation-Files?v=t):
+* BIOS Binary/Capsule: capsule-<BIOS Version>-r0.core2_64.rpm
```
Comment (lucasdemarchi) [inline docs/release/release-notes.md]: BIOS is now embedded in the image

File: docs/release/release-notes.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+* OS Image : intel-aero-image-intel-aero-<OS Version>.iso
+
+Please check [here](https://github.com/intel-aero/meta-intel-aero/wiki/02-Initial-setup#flashing) for upgrade process
```
Comment (lucasdemarchi) [inline docs/release/release-notes.md]: Link should be fixed to point to the right place

**Comment by anitha-suresh** [comment]
@lucasdemarchi , I initially thought that it would be part of 'Software' tab. However, do you think we should have all the technical information in that tab and Releases as a separate one.

**Comment by avinash-palleti** [comment]
@lucasdemarchi Please check now. I have added into software tab, removed BIOS line and added right link for update procedure.

**Comment by lucasdemarchi** [comment]
> @lucasdemarchi , I initially thought that it would be part of 'Software' tab. However, do you think we should have all the technical information in that tab and Releases as a separate one.

I think it should be in Software.  But more importantly it should be **somewhere**.  I fetched the PR to check everything was ok just to realize it was not showing anywhere. And that it was missing the integration in mkdocs.   That makes me wonder how that PR could ever had been tested.  Untested code without a big warning saying it's still untested makes me grumpy.

**Comment by lucasdemarchi** [comment]
And it still doesn't render correctly. Please see the README file: you need basically 2 commands:

```
make setup # just once
make serve
```

**Comment by avinash-palleti** [comment]
@lucasdemarchi Missed "make serve" step but i verified it after pushing into my branch, saw it was rendering properly as below then only submitted PR. Now realized it is not rendering when i do "make serve". I will update the PR.

untitled

**Comment by avinash-palleti** [comment]
@lucasdemarchi updated commit and verified it with "make serve" locally.

**Comment by avinash-palleti** [comment]
Now realized that there is difference in "make serve" step and looking at md file in github. Will take care next time :)

**Comment by lucasdemarchi** [comment]
Thank you, pull request merged! [See changes here.](https://github.com/intel-aero/intel-aero-docs/compare/6069b10~1...6069b10)

**Pull request closed without merging**